### PR TITLE
remove usage of @apply and @screen

### LINKED ISSUE #2012: refactor css
Unfortunately, there is no way of using styled-jsx with postcss without relying on babel -> this is a opt-out for the new swc compiler. The only way in the meanwhile is to remove unsupported features from styles:

### tailwind `@apply`

- set classname directly or
- use arbitrary selector to set style on multiple elements

### nesting

- use css modules?
- manually unnest
- ...?

I would like to finish this in the next 1 - 2 month, if we work on this now and then it should be doable. I can take care of the tailwind stuff, @elbotho can you take a look at the nesting situation? we would also like to use turbopack, and they have postcss support (https://nextjs.org/blog/next-13-1#turbopack-improvements)

### PR DESCRIPTION
**Pull request by Entkenntnis**
first step in the css refactoring

remove all usages of @apply and @screen

### CODE REVIEW

File: src/edtr-io/plugins/types/helpers/revision-history-loader.tsx [hunk 1]
```diff
@@ -85,12 +85,18 @@ export function RevisionHistoryLoader<T>(
       </ModalWithCloseButton>
       <style jsx global>{`
         .ReactModalPortal .ReactModal__Content {
-          @apply w-[900px] max-w-[90vw] max-h-[80vh];
-          @apply translate-y-0 top-8 overflow-y-scroll;
-          @apply pt-0;
+          /* this css colides with save-modal.tsx */
```
Comment (Entkenntnis) [inline src/edtr-io/plugins/types/helpers/revision-history-loader.tsx:88]: I haven't refactored this, but instead kept it as is

File: src/helper/breakpoints.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const breakpoints = {
```
Comment (elbotho) [inline src/helper/breakpoints.ts:2]: those are straight from tailwind, right? 
Maybe we should add that in a short comment.

File: src/edtr-io/components/save-modal.tsx [hunk 3]
```diff
@@ -226,17 +231,10 @@ export function SaveModal({
           }}
         />{' '}
         <span
-          className="license-wrapper"
+          className="license-wrapper [&_a]:!text-brand hover:[&_a]:underline"
```
Comment (elbotho) [inline src/edtr-io/components/save-modal.tsx:234]: this syntax hurts a bit to be honest :D 
But if it works…

**Review by elbotho — APPROVED** [review]
Thanks for starting this! 👍
If you checked the visuals and mobile views for regressions I approve :)

File: src/edtr-io/components/save-modal.tsx (hunk 3 above)
Comment (Entkenntnis) [reply src/edtr-io/components/save-modal.tsx:234]: it works :)

File: src/helper/breakpoints.ts (hunk 2 above)
Comment (Entkenntnis) [reply src/helper/breakpoints.ts:2]: ✔️

**Pull request merged**